.gitignore

```diff
@@ -9,3 +9,5 @@ packages/**/LICENSE
 temp
 tmp-*
 /_site/
+packages/workbox-webpack-plugin/**/webpack-assets/sw*
```

Comment:
could you use a `tmp-*` file name if you are dynamically creating folders in tests?